helpers/control.py

```diff
@@ -322,6 +322,7 @@ def __init__(self, vhost_auto=True):
         self.host = tf_cfg.cfg.get("Tempesta", "hostname")
         self.err_msg = " ".join(["Can't %s TempestaFW on", self.host])
         self.stop_procedures = [self.stop_tempesta, self.remove_config]
+        self.__check_config = True
```

Comment:
I try to limit any interactions with class variables from the outside. Probably we can use setter here but I don't think that we need to enable check config because we must always check TempestaFW config. This case is very rare